### DISCUSSION TITLE
Unable to get BG from xdrip to nightscout on railway

### DISCUSSION THREAD

**Discussion by stevedeichmann62** [NightscoutFoundation/xDrip#2869] [General]
***SOLVED***
I want my BG to go to nightscout on railway. All other data goes, but not the BG.

This is the text from the uploader status page in xdrip+.

Any ideas?

Libre2

Uploaders

Cloud Uploader Queues

Nightscout REST

: 2087 BgReading

Last poll: 3 minutes ago

REST-API problem

: Unable to do REST

2023-06-01 20:53:47 (2)

API Upload: Attempt th(java.lang.String)' on

to invoke virtual method 'boolean java.lang.String.startsWi

a null object reference marking record: failed

**Comment by Navid200** [comment]
The best person who can help you is someone else using railway.
You can find them on Nightscout support groups.

This shows xDrip's uploader syntax for Nightscout:
https://navid200.github.io/xDrip/docs/Nightscout-Settings.html

**Reply by stevedeichmann62** [reply]
SOLVED
I had a space before my password. Thank you!
